### DISCUSSION TITLE
zsh: command not found: git_current_branch

### DISCUSSION THREAD

**Discussion by felinae98** [zimfw/zimfw#501] [Q&A] [answered]
Hello, I'm using zimfw and git plugin of omz. The alias `gpsup` alias calls `git_current_branch`, which is proved in omz, but not in zimfw. Any ideas to solve this issue?

**Comment by ericbn** [comment, marked as answer]
Looks like you'll also have to add
```
zmodule ohmyzsh/ohmyzsh --source lib/git.zsh
```
to your .zimrc, as the `git_current_branch` and other git-related functions in ohmyzsh reside there.
